Funda mapjes - kopie/styles/nieuwefundastyle.css

```
@@ -0,0 +1,342 @@
+:root{
+    --primaire-color: #F7A100;
+    --secundaire-color: #0071B3;
+    --primaire-lettertype: Proxima Nova;
+    --navbarlettertype: 15px;
+}
+
+.maandlasten {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: proxima nova, sans-serif;
+}
+
+/* BEGIN HEADER */
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    /* width: 414px; */
+    height: 60px;
+    border: 0px;
+    top: 0px;    
+    background-color: var(--primaire-color);
+}
```

Ik vraag me af of sommige van deze eigenschappen wel gebruikt worden. Ik zou geen fixed height `60px` gebruiken voor de header, maar de hoogte bepalen aan de hand van de `font-size`, `line-height` en `padding` eigenschappen. Op deze manier schaalt het element mee wanneer de gebruiker het zoomniveau aanpast. Mocht je dan toch willen dat het element niet meeschaalt kun je er voor kiezen om een vaste `font-size` in `px` te gebruiken.

```suggestion
nav {
    display: flex;
    justify-content: space-between;
    height: 60px;
    background-color: var(--primaire-color);
}
```